src/test/java/dk/kb/present/transform/XSLTPreservicaToSolrTransformerTest.java

```diff
@@ -65,7 +65,23 @@ public void testCalculatedDuration() {
 
     @Test
     public void testGenrePresent() {
-        assertContains(RECORD_44979f67, "\"genre\":[\"");
+        assertContains(RECORD_44979f67, "\"genre\":\"");
+    }
+
+    @Test
+    void testGenreContent() {
+        assertContains(RECORD_a8aafb121,"\"list_of_categories\":[\"hovedgenre: Serier\",\"undergenre: Krimiserie\"");
+        assertNotContains(RECORD_a8aafb121, "\"list_of_categories\":[\"hovedgenre: Serier ritzau\",\"undergenre: Krimiserie ritzau\"");
```

Comment:
The `hovedgenre: Serier` notation (not a big fan of the "implicit datastructure hidden in a plain string BTW, but I see the challenge here) is at odds with the examples in the `list_of_categories` field in the Solr `schema.xml`.